doc/whats_new.rst

```diff
@@ -108,6 +108,18 @@ Enhancements
    - Added ``inverse_transform`` function to :class:`decomposition.nmf` to compute
      data matrix of original shape. By `Anish Shah`_.
 
+   - Enhanced MLP with Adams, Nesterov's momentum, and early stopping
+     (`#5214 <https://github.com/scikit-learn/scikit-learn/pull/5214>`_) by `Jiyuan Qian`_
```

Comment:
Is this the only entry for MLPs? That was the work of issam, me, olivier and Jiyuan, I think. Also, maybe if this is the first entry for an MLP, it shouldn't be called "Enhanced MLP".